test/project_types/theme/commands/pull_test.rb

```diff
@@ -8,6 +8,7 @@ class PullTest < MiniTest::Test
 
       def setup
         super
+        project_context("theme")
```

Comment:
By default, cwd is `test/fixtures/project`.
In order to work in a _valid_ _theme_ directory, I have used `project_context`, which includes a `cd` into `test/fixtures/theme`.
I initially tried with setting the `root` parameter (at multiple places), but it kept staying in `test/fixtures/project`.
Even though I don't like changing dirs during tests, I think it's a cleaner solution than explicitly setting `root` everywhere (provided that this would work).

Reply:
> project_context("theme") doesn't have a clear function without exploring the meaning of that method

That's true. I trusted/got inspired by the existing code where `project_context` was used at least 7 times 😄.

> will that be super clear when we come back to this code in 6 months?

Probably not, but I accepted this, as I thought we would migrate the existing theme code to 3.x sooner 😄. But I will try again with setting root in the tests then :)